static/js/clip.js

```diff
@@ -22,4 +23,18 @@
             return trigger.previousElementSibling;
         }
     });
-})(document, Clipboard);
\ No newline at end of file
+})(document, Clipboard);
+
+
+/*
+* sends clicks on clipboard buttons with .copy class as Google Analytics events
+*/
+
+(function() {
+    const pageURL = document.location.pathname + document.location.search;
+
+    var button = document.getElementsByClassName("copy");
+    button.addEventListener("click", function() {
+        ga('send','event','Clipboard','Code copied',pageURL);
+    });
```

Comment:
```suggestion
    var buttonSet = document.querySelectorAll(".copy");
    buttonSet.forEach(function (btn) {
      btn.addEventListener("click", function() {
        ga('send','event','Clipboard','Code copied',pageURL);
       });
     });
```

I don't think this is going to work, afaik `getElementsByClassName` returns an `HTMLCollection` (a _list_ of nodes). As such we'll need to iterate over them and bind listeners to each.

Reply:
Updated suggestion to include guard for case where google analytics is missing.